sopel_modules/weather/wz.py

```diff
@@ -53,13 +54,13 @@ def _get(self, text):
 
         location = self.here.location(text)
 
-        gps_loc = location["Location"]["NavigationPosition"][-1]
-        city = location["Location"]["Address"]["City"]
-        state = location["Location"]["Address"]["State"]
+        gps_loc = location["position"]
+        city = location["address"]["city"]
+        state = location["address"]["stateCode"]
 
```

Comment:
if 'city' in location["address"] and 'stateCode' in ocation["address"]:
  address = f"{location["address"]['city'], location["address"]['stateCode']"
else:
  address = f"{location["address"]["label"]